root/search/components/SeriesResults.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../../context';
+import {l} from '../../static/scripts/common/i18n';
+import {l_attributes} from '../../static/scripts/common/i18n/attributes';
+import PaginatedResults from '../../components/PaginatedResults';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import loopParity from '../../utility/loopParity';
+import type {ResultsPropsT} from '../types';
+
+import ResultsLayout from './ResultsLayout';
+
+function buildResult(result, index) {
+  const series = result.entity;
+  const score = result.score;
+
+  return (
+    <tr className={loopParity(index)} key={series.id}>
+      <td>{result.score}</td>
+      <td>
+        <EntityLink entity={series} />
+      </td>
+      <td>
+        {series.typeName ? l_attributes(series.typeName) : null}
```

Comment:
Same as https://github.com/metabrainz/musicbrainz-server/pull/693#discussion_r201277122, context is missing for localization, should be:
```js
lp_attributes(series.typeName, 'series_type')
```